pkg/git/fetch_test.go

```diff
@@ -89,26 +79,55 @@ func testGitCheckout(t *testing.T, when spec.G, it spec.S) {
 
 		it("returns error on non-existent ref", func() {
 			err := fetcher.Fetch(testDir, "https://github.com/git-fixtures/basic", "doesnotexist", metadataDir)
-			require.EqualError(t, err, "could not find reference: doesnotexist")
+			require.EqualError(t, err, "resolving revision: reference not found")
 		})
 
-		it("returns error from remote fetch when authentication required", func() {
+		it("preserves symbolic links", func() {
+			err := fetcher.Fetch(testDir, "https://github.com/git-fixtures/symlinks", "master", metadataDir)
+			require.NoError(t, err)
+			fileInfo, err := os.Lstat(path.Join(testDir, "bar"))
```

Comment:
nit: spacing is important for readbility
```suggestion
			require.NoError(t, err)
			
			fileInfo, err := os.Lstat(path.Join(testDir, "bar"))
```

Reply:
👍 This and [more](https://github.com/pivotal/kpack/pull/1159/commits/fccb5a7edae5e17b2bdefa370ff277450961a47d#diff-b340348c7f8c956d7b67e307a714ca4bb3077f355e44c643d45e534b1ad7e7d3R88)